src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/incidents/IncidentResource.kt

```diff
@@ -141,8 +206,8 @@ data class IncidentResponse(
   val title: String?,
   @Schema(description = "The incident details")
   val description: String?,
-  @Schema(description = "Prison where the incident occurred")
-  val prison: CodeDescription,
+  @Schema(description = "Agency Location where the incident occurred")
+  val location: CodeDescription,
```

Comment:
I hate to say it but maybe 'location' seems a bit too easy to confuse with locations (as in within a prison), maybe something like 'agency' and agencyId instead of prisonId is better?

Reply:
funny- I toyed with this - I will change to agencyId in separate PR - I found an agencyLocation that wasn't a prison (it was PECS) so  leaving as prison didn't make sense anymore.